src/app.js

```diff
@@ -646,6 +646,8 @@ var run = function () {
     startdebugging(txResult.transactionHash)
   })
 
+  $('#createAccount').click(function () { udapp.newAccount() })
```

Comment:
The UI must be refreshed (to populate the account dropdown) after a new account is created.